### PR TITLE
[IMP] barcodes_generator_product: OpenUpgrade migration script

### PR DESCRIPTION
**Pull request by pedrobaeza**
Script for migrations coming from v8 module product_barcode_generator

cc @Tecnativa

### CODE REVIEW

File: barcodes_generator_product/migrations/9.0.1.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    """This converts the sequences found with a barcode to barcode rules for
+    being selectable at product level."""
+    if not openupgrade.column_exists(
+            env.cr, 'ir_sequence', 'barcode_sequence'
+    ):
+        # Return if product_barcode_generator was not previously installed
+        return
+    rule_obj = env['barcode.rule']
+    env.cr.execute("""
+        SELECT id, name, barcode_sequence
```
Comment (chienandalu) [inline barcodes_generator_product/migrations/9.0.1.0.0/post-migration.py:18]: Shouldn't `number_next_actual` be fetched?

Comment (pedrobaeza) [reply barcodes_generator_product/migrations/9.0.1.0.0/post-migration.py:18]: No, this relies the same on the sequence, so no need of that.

Comment (chienandalu) [reply barcodes_generator_product/migrations/9.0.1.0.0/post-migration.py:18]: Yes, I see.

**Review by chienandalu — APPROVED** [review]

**Review by luismontalba — APPROVED** [review]

**Comment by pedrobaeza** [comment]
OK, merging then.

**Pull request merged**